src/Progress/LinearProgress.js

```diff
@@ -8,38 +8,56 @@ import withStyles from '../styles/withStyles';
 
 const transitionDuration = 4; // 400ms
 
+const getBar = color => ({
```

Comment:
Just updated now.  The only redundancy is that the background attributes for the color-specific dashed classes.